### PR TITLE
Updated several changes effective in Rails 5.0

### PR DESCRIPTION
**Pull request by alxcrrll**
Just need to ensure that config vars ENV['SECRET_KEY_BASE'] is set before merging.

### CODE REVIEW

File: config/environments/staging.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  # config.autoflush_log = false
+
+  # Use default logging formatter so that PID and timestamp are not suppressed.
+  config.log_formatter = ::Logger::Formatter.new
+
+  # ActionMailer Config
+  config.action_mailer.default_url_options = { host: ENV['ACTION_MAILER_HOST'] }
+  # Setup for production - deliveries, no errors raised
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = (ENV['ACTION_MAILER_PERFORM_DELIVERIES'] == 'true')
```
Comment (mhudson) [inline config/environments/staging.rb:115]: I think staging _should_ perform deliveries, but they should be re-routed to a dummy mailbox (as you suggested)

File: config/environments/staging.rb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  config.assets.compile = false
+
+  # Add the fonts path
+  config.assets.paths << Rails.root.join('app', 'assets', 'fonts')
+
+  # Precompile additional assets
+  config.assets.precompile += %w[
+    .svg .eot .woff .ttf
+
+    admin-layout.css admin-layout.js
```
Comment (mhudson) [inline config/environments/staging.rb:41]: Does `precompile` compile `.css` and `.js` by default?

File: app/views/layouts/mailer.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (mhudson) [inline app/views/layouts/mailer.html.erb:1]: What's this file for? Just a default?

Comment (alxcrrll) [reply app/views/layouts/mailer.html.erb:1]: Yes, this is a  default template layout.

File: config/environments/staging.rb (hunk 2 above)
Comment (alxcrrll) [reply config/environments/staging.rb:41]: This probably best explains it. We could certainly refactor this though.

https://guides.rubyonrails.org/asset_pipeline.html#precompiling-assets



Screen Shot 2021-08-13 at 5 42 52 pm

File: config/environments/staging.rb (hunk 1 above)
Comment (alxcrrll) [reply config/environments/staging.rb:115]: I agree. I have looked at a couple of options to achieve the re-routing outcome but haven't found something that suits our workflow and budget.

Comment (mhudson) [reply config/environments/staging.rb:115]: Ok, as long as we _never_ put the production database on staging.

**Review by mhudson — APPROVED** [review]

**Pull request merged**